pybikes/bysykkel_graphql.py

```diff
@@ -0,0 +1,90 @@
+# -*- coding: utf-8 -*-
+import json
+
+from . import utils
+from .base import BikeShareStation, BikeShareSystem
+
+BYSYKKEL_GRAPHQL_URL = "https://core.urbansharing.com/public/api/v1/graphql"
+
+
+class BysykkelGraphql(BikeShareSystem):
+
+    authed = False
+
+    meta = {
+        'system': 'Bysykkel GraphQL',
+        'company': ['Urban Infrastructure Partner'],
+    }
+
+    def __init__(self, tag, meta):
+        super(BysykkelGraphql, self).__init__(tag, meta)
+        self.feed_url = BYSYKKEL_GRAPHQL_URL
```

Comment:
Once you get the `systemId` here you can leave BYSYKKEL_GRAPHQL_URL outside the class (see comment about the json data file).

Reply:
Aha, should be good to go now.